src/rules/noTrailingWhitespaceRule.ts

```diff
@@ -87,7 +89,14 @@ function walk(ctx: Lint.WalkContext<Options>) {
     for (const line of getLineRanges(sourceFile)) {
         // \s matches any whitespace character (equal to [\r\n\t\f\v ])
         const match = text.substr(line.pos, line.contentLength).match(/\s+$/);
-        if (match !== null && !(ctx.options.ignoreBlankLines && match.index === 0)) {
+        if (
+            match !== null &&
+            !(ctx.options.ignoreBlankLines && match.index === 0) &&
+            // See https://github.com/palantir/tslint/issues/4518
+            // and https://github.com/ajafff/tsutils/issues/94
+            // TODO: Remove this check once tsutils handles Byte Order Markers correctly.
```

Comment:
You can remove this TODO; see discussion in the tsutils issue.

Reply:
Got it!